specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -369,6 +369,167 @@
           "application/json"
         ]
       }
+    },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/providers/Microsoft.GuestConfiguration/softwareUpdateProfile": {
+          "get": {
+            "tags": [
+              "SoftwareUpdateProfile"
+            ],
+            "operationId": "SoftwareUpdateProfile_Get",
+            "description": "Get the singleton software update profile for the computer.",
+            "x-ms-examples": {
+              "Get the singleton software update profile for a computer.": {
+                "$ref": "./examples/getSoftwareUpdateProfile.json"
+              }
+            },
+            "parameters": [
+              {
+                "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+              },
+              {
+                "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+              },
+              {
+                "$ref": "../../common/v1/definitions.json#/parameters/VmNameParameter"
+              },
+              {
+                "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+              },
+              {
+                "name": "softwareUpdateId",
```

Comment:
This parameter does not exist in the path.

Reply:
Fixed